tests/integration/s3/test_s3_cors.py

```diff
@@ -109,9 +112,9 @@ def test_cors_http_get_no_config(self, s3_bucket, snapshot, aws_client):
         response = aws_client.s3.put_object(Bucket=s3_bucket, Key=key, Body=body, ACL="public-read")
         assert response["ResponseMetadata"]["HTTPStatusCode"] == 200
 
-        # key_url = f"{_bucket_url_vhost(bucket_name=s3_bucket)}/{key}"
+        key_url = f"{_bucket_url_vhost(bucket_name=s3_bucket)}/{key}"
```

Comment:
Are the changes here in the file some leftover from debugging or intentional? If they are intentional, I think the comment should be removed.

Reply:
Oh, those were a remnant from the CORS issue linked to thread local, and I think Dominik changed it again to be able to run against AWS. I guess all of those could be set back to use `_bucket_url_vhost` and remove all the comments. I can do in a subsequent PR as well

Reply:
@bentsku already kinda explained how that slipped in

will revert :+1: